o2a/converter/oozie_converter.py

```diff
@@ -28,6 +28,7 @@
 from o2a.converter.property_parser import PropertyParser
 from o2a.converter.relation import Relation
 from o2a.converter.renderers import BaseRenderer
+from o2a.converter.tasks_group import TaskGroup
```

Comment:
One small suggestion for name - we could simply use Tasks (but it's just a suggestion - TaskGroup is fine as well)

Reply:
Tasks is confusing to me. We often have variables called "tasks" when we talk about a list of tasks. I prefer to avoid the creation of classes in the plural form.

Reply:
It will also be difficult to create a list of such objects when we already have a plural in the class name.
```
task_groups = [TaskGroup(), TaskGroup(), TaskGroup()]
```
```
? = [Tasks(), Tasks(), Tasks()] 
```
What should the variable be called?

Reply:
I like `TaskGroup`. Maybe I'd suggest renaming the module from `.tasks_group` to `.task_group`, WDYT?

Reply:
I will correct the file name.

Reply:
TaskGroup is fine :)

Reply:
I changed the file name.